### PR TITLE
Add configuration to support PARAMETER sole attribute

### PR DESCRIPTION
**Pull request by Pranavchiku**
Separate PR for ```Parameter``` #669

### CODE REVIEW

**Comment by certik** [comment]
I think that looks good. Make sure tests pass and then ping me and I'll do a final review. Thanks for fixing this!

**Comment by Pranavchiku** [comment]
@certik you can have a final review :)

File: tests/reference/asr-parameter1-1a7ed3b.stderr [hunk 1]
```diff
@@ -0,0 +1,5 @@
+semantic error: Attribute declaration not supported
+ --> tests/parameter1.f90:3:1
+  |
+3 | parameter (zero=0.0d0,p66=0.66d0,two=2.0d0,three=3.0d0)
+  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```
Comment (certik) [inline tests/reference/asr-parameter1-1a7ed3b.stderr:5]: I think this file should not be here?

**Comment by certik** [comment]
Do:
```
rm -rf tests/reference/*
./run_tests -u
git add tests/reference
```
I think it will remove one unused reference (error) file.

**Comment by Smit-create** [comment]
These changes are already in #675.

**Comment by Pranavchiku** [comment]
Crap! yeah I didn't see that one, should I close this and #684 then?

**Comment by Pranavchiku** [comment]
Closed in reference to #675

**Pull request closed without merging**